kast-json.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
Is this intended? On some systems where both `python2` and `python3` this can be a problem.